### ISSUE TITLE
Error when entering Tuesday afternoon price

### ISSUE THREAD

**Issue by BaNkIck** [mikebryant/ac-nh-turnip-prices#431]
**Things to check**
- Have you forced a refresh in your browser (e.g. ctrl+F5 in Chrome/Firefox)?
  - yes
- Have you made sure the first time buyer option is correct? If you're unsure, try it both ways
  - yes
- Have you time-travelled this week? Travelling backwards resets the prices, and therefore you must not put prices in from both before & after the time-travel
  - no

**Describe the bug**
When entering the price of Tuesday afternoon, an error appears saying to report it

**To Reproduce**
1. Enter Sunday price (94)
2. Enter prices of Monday and Tuesday morning (121, 97, and 134 respectively)
3. Enter the price of Tuesday afternoon (114)

**Permalink to your prices**
https://turnipprophet.io?prices=94.121.97.134.114........&first=true

**Expected behavior**
See the prediction graph and table.

**Screenshot**
![image](https://user-images.githubusercontent.com/843170/102241204-fc431b80-3ef8-11eb-96bb-4e9a3a5a01bd.png)

**Desktop (please complete the following information):**
 - OS: Mac OS Catalina (10.15.7)
 - Browser: Chrome (Versió 87.0.4280.88 (Muntatge oficial) (x86_64))

**Comment by Kotickiha** [comment]
Same issue, but I can't enter any price after monday.

**Comment by peter50216** [comment]
Have you made sure the first time buyer option is correct?

There's prediction after changing the first time buyer option to "no".

**Comment by BaNkIck** [comment]
Yes, it was my first time buying so that option is intentionally set to "yes".
